### PR TITLE
NEWS: version 3.23

### PR DESCRIPTION
**Pull request by MrAnno**
Pending:
- [x] Add contextual data make filters config plugin aware (#2886)
- [x] Fix minor resource/memory leaks, null pointer dereference (#2875)

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: NEWS.md [hunk 1]
```diff
@@ -1,73 +1,62 @@
...
+ * Add global context to standalone parsers (#2876)
+ * Fix heap usage after free in cfg_run_parser_with_main_context (#2884)
+ * Fix g_thread_init call order issue with older glibs (#2853)
+ * SNMP destination: fixing statistics format (#2854)
+ * eventlog: flush escaped_buffer when full (#2837)
+ * dbparser: remove unnecessary lock (#2838)
+ * dbparser: fix crash when context times out in the middle of another rule (#2832)
+ * radix: fix grouping in PCRE (#2808)
+ * add-contextual-data: make filters config plugin aware (#2886)
+ * Fix distro packages build (#2883)
```
Comment (gaborznagy) [inline NEWS.md]: IMHO no need to announce this. Fixed a regression.

File: NEWS.md [hunk 2]
```diff
@@ -1,73 +1,62 @@
...
- * `http()`: Warn if less workers used than urls (#2757)
- * `http()`: Allow URLs to be specified by a space/comma separated string
-   (#2699)
- * loggen: Change message rate at runtime using signals (#2756)
- * debun: add acquire_running_syslog_config function (#2752)
- * FreeBSD fixes for the test suite (#2783)
+ * Disable `time-reap()` on non-templated filenames by default (#2798)
+ * The `--preprocess-into` command line flag accepts `-`, and writes the
+   preprocessed configuration to stdout. (#2869)
+ * Remove POSIX regex support (part 2) (#2849)
```
Comment (gaborznagy) [inline NEWS.md]: Also, no need to announce this, 
especially with a naming that suggests that further POSIX regex support has been removed.

File: NEWS.md (hunk 1 above)
Comment (MrAnno) [reply NEWS.md]: Indeed, thanks!

File: NEWS.md (hunk 2 above)
Comment (MrAnno) [reply NEWS.md]: Thanks.

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by MrAnno** [comment]
@kira-syslogng retest this please

File: NEWS.md [hunk 3]
```diff
@@ -1,73 +1,60 @@
...
 
-## Features
+ * Add relocation support for `disk-buffer()`. The `relocate` subcommand of
+   `dqtool` can be used to move a single or multiple queue files. (#2855)
 
- * `match()`: Add support for the `template()` option (#2715)
- * `add-contextual-data()`: Allow using templates in name-value pairs (#2711)
- * Add support for floating point operations in template functions (#2742)
- * Add support for usec precision when parsing time (#2709)
+ * `file(), pipe()`: The `time-reap()` option now can be set or disabled for
```
Comment (alltilla) [inline NEWS.md]: I would also notify the users, that for non-templated sources, time-reap is now disabled by default.

**Comment by kira-syslogng** [comment]
Build FAILURE

File: NEWS.md (hunk 3 above)
Comment (MrAnno) [reply NEWS.md]: It has a separate entry in `Other changes`.

**Comment by MrAnno** [comment]
@kira-syslogng retest this please

File: NEWS.md (hunk 3 above)
Comment (alltilla) [reply NEWS.md]: Oh my bad! Thanks! I also meant destination, don't mind me :D

**Comment by MrAnno** [comment]
@kira-syslogng retest this please

**Comment by MrAnno** [comment]
@kira-syslogng retest this please

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by MrAnno** [comment]
@kira-syslogng retest this please

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**